diffblue-base.yml

```diff
@@ -8,5 +8,7 @@
 # merging a custom yml file on top of this one using the script
 # in dashboard/scripts/update-diffblue-yml. Any options specified here would
 # be overriden to keep consistency with other benchmarks.
-buildCmd: cd src/supernode; mvn compile
+# mvn -X adds debugging info to the build output which is necessary to build
+# the classpath.
```

Comment:
:pick: Could mention that `mvn -e -X -Dmaven.test.skip=true compile` is the default build command and the only difference is the `cd` command.

Reply:
Yes, that's what I meant actually :slightly_smiling_face: